pycpfcnpj/calculation.py

```diff
@@ -20,19 +20,20 @@ def calculate_first_digit(number):
 
     sum = 0
     if len(number) == 9:
-        weights = CPF_WEIGHTS[0]
+        weights: tuple = CPF_WEIGHTS[0]
     else:
-        weights = CNPJ_WEIGHTS[0]
+        weights: tuple = CNPJ_WEIGHTS[0]
 
     for i in range(len(number)):
-        sum = sum + int(number[i]) * weights[i]
-    rest_division = sum % DIVISOR
+        sum: int = sum + int(number[i]) * weights[i]
```

Comment:
Sorry for the super late response. I'm ok with most of the annotations, but did you try mypy first? 
Try to install it locally `pip install mypy` and execute it on each file, like `mypy pycpfcnpj/calculation.py` and check a few issues. 

This one, for instance, you should put, although I don't care much, the type on the first `sum` when it's first declared. 

For the `weights` variable, a solution is use a single lined IF, like below: 
`weights: tuple = CPF_WEIGHTS[0] if len(number) == 9 else CNPJ_WEIGHTS[0]`

Ping me if you have any further questions.